packages/compartment-mapper/src/node-modules.js

```diff
@@ -636,7 +636,7 @@ const translateGraph = (
               name,
               path,
             },
-            packagePolicy,
+            /** @type {import('./types.js').PackagePolicy} */ (packagePolicy),
```

Comment:
note:
When we rebase this on top of #1838 the `packagePolicy` is always an object if `policy` is truthy. 
Currently passing undefined is being prevented by an early `continue`  on line 617